api/node/node.go

```diff
@@ -35,5 +35,6 @@ func (n *Node) Mount(root *mux.Router, pathPrefix string) {
 
 	sub.Path("/network/peers").
 		Methods(http.MethodGet).
-		HandlerFunc(utils.MetricsWrapHandlerFunc(pathPrefix, "node_network", n.handleNetwork))
+		Name("node_network_peers").
+		HandlerFunc(utils.WrapHandlerFunc(n.handleNetwork))
```

Comment:
Would be good to keep the name as close as possible to the URL and avoid the interpretation step.
Ie. `curl .../network/peers` -> `network_get_peers`

Reply:
I would go for `node_get_network_peers` since it is under the path `node`